app/lib/humane_transport_app.dart

```diff
@@ -1,10 +1,22 @@
 import 'package:app/screens/home/home.dart';
+import 'package:app/style/style.dart';
 import 'package:flutter/material.dart';
 
 class HumaneTransportApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
       home: HomePage(),
+      // This is one way to consume styling properties inside style.dart
```

Comment:
Not sure this comment needs to stay in the code. If you intend for this to be an FYI for reviewers, you can comment on your code in your PR for us to see.